src/gui/tool/tool_SRTPanel.cpp

```diff
@@ -41,30 +41,30 @@ void SRTPanel::createMode()
         this->onParamUpdated(true);
     });
 
-    mAddMove.reset(new CheckBoxItem(tr("necessarily move"), this));
-    mAddMove->setToolTip(tr("Necessarily add a MoveKey if the posture is modified."));
+    mAddMove.reset(new CheckBoxItem(tr("Always move"), this));
+    mAddMove->setToolTip(tr("Always add a move key when modifying the posture."));
```

Comment:
I was unsure between the differences regarding pose and posture in the code, since as far as I remember, the original Japanese translation mentioned both, I couldn't understand the entirety of the code to decide on the better one myself so I would love some feedback on this.